docs/references/v2.openapi.json

```diff
@@ -717,10 +717,19 @@
             "$ref": "#/components/responses/422"
           }
         },
-        "description": "This endpoint enables the broadcast of an event to a room without having to connect to it via the `client` from `@liveblocks/client`.",
+        "description": "This endpoint enables the broadcast of an event to a room without having to connect to it via the `client` from `@liveblocks/client`. It takes any valid JSON as a request body.",
         "requestBody": {
           "content": {
-            "application/json": {},
+            "application/json": {
+              "examples": {
+                "example-1": {
+                  "value": {
+                    "jobId": "process-image",
+                    "status": "completed"
```

Comment:
```suggestion
                    type: "EMOJI", emoji: "🔥"
```

Reply:
Could we use this emoji example from the rest of the docs (e.g. [here](https://liveblocks.io/docs/api-reference/liveblocks-react#useBroadcastEvent)). Consistency is good for our users.

Reply:
Hah! That's what I went for originally, of course!